### ISSUE TITLE
Frostfire Mage Sim discrepency Desktop vs Web

### ISSUE THREAD

**Issue by John20001** [wowsims/wotlk#2820]
**Describe the Problem**
I downloaded the windows.exe version. I imported the same mage gear/gems. Talents are the same, stats are the same, debuffs are apparently the same though that is my suspicion if anything. I'll provide screenshots to compare:

**Desired Solution**
It's a minor discrepency but would like to help you find the bug / error (user or otherwise). 

I will produce image pairs, where the first is the web app, and the second is my local app. Happy to provide logs if I can figure out how. 

![image](https://user-images.githubusercontent.com/6766918/227613361-2320e29d-cf16-4acc-91af-bd95256b1381.png)
![image](https://user-images.githubusercontent.com/6766918/227613381-d9df5766-d5bd-43ac-a77e-6075cbfbdd13.png)

![image](https://user-images.githubusercontent.com/6766918/227613483-f701374b-f2a9-4482-9f5b-07201dfbd0a7.png)
![image](https://user-images.githubusercontent.com/6766918/227613522-a230622c-3d91-47ec-9100-364ab99db2ca.png)

![image](https://user-images.githubusercontent.com/6766918/227613560-637e89d6-fc6c-4ba5-861e-ae723400e02c.png)
![image](https://user-images.githubusercontent.com/6766918/227613591-577206cd-9d82-4639-ac3e-051d5bbb20a3.png)

![image](https://user-images.githubusercontent.com/6766918/227613744-81dfc9e9-28d3-4207-b355-8ee00270758b.png)
![image](https://user-images.githubusercontent.com/6766918/227613781-6624834c-56b1-40f5-8579-72730ed40c00.png)

This appears to be a consistent difference of about 100 dps, or 1.2%, so im not sure what that would be. If there is any other information I can provide, please let me know. Thank you guys for this awesome tool!

**Comment by John20001** [comment]
One hint is that the crit seems stable, but the average damage itself is higher. This leads me to believe its a possible issue in the settings / debuffs. But they all look the same to me?

**Comment by lime-green** [comment]
@John20001 i just fixed the mage t8 buff, so it could be that since it would on the web version but not yet on the released executable. You can check the buff uptime, it’s called Praxis

**Comment by John20001** [comment]
@lime-green Yep! No Praxis on executable, but Praxis on web. Will be awaiting the next release, thanks for the quick response!
